torch/onnx/symbolic_opset9.py

```diff
@@ -3927,6 +3927,9 @@ def linspace(g, start, end, steps, dtype, layout, device, pin_memory):
     )
     return add(g, mul(g, range_tensor, step), start)
 
+def lift(g, self):
+    # at::lift() is a no-op from the perspective of tracing for onnx
+    return self
```

Comment:
After reading through some onnx docs / looking at previous PR's, I added this and was able to get the failing onnx tests to pass locally. (I think a no-op is fine since lifting should be traced away before you create a graph)

Reply:
(this is the only change on this PR compared to the original)